Source/WebKit/Platform/IPC/Connection.h

```diff
@@ -208,6 +208,9 @@ class Connection : public ThreadSafeRefCountedAndCanMakeThreadSafeWeakPtr<Connec
     public:
         virtual void didClose(Connection&) = 0;
         virtual void didReceiveInvalidMessage(Connection&, MessageName) = 0;
+#if USE(EXTENSIONKIT)
+        virtual void didRequestProcessTermination(bool& wasTerminated) { wasTerminated = false; }
```

Comment:
Given that we're asking the client to terminate, I think we should use a verb in the name:
e.g. "requestProcessTermination()"

I also think it should return the boolean instead of using an old-style out-parameter.

Reply:
Also, could this be pure virtual? Seems risky for a subclass to NOT override this.

Reply:
That is a good point. I did not add this in the latest patch, since only AuxiliaryProcessProxy is calling `Connection::kill()`, currently. However, please let me know if you still think we should make it pure virtual, I will upload a new PR with that change :)

Thanks for reviewing!

Reply:
Yes, I think it should be pure virtual.. You only provided evidence that it SHOULD be pure virtual so why wouldn't we want it to not be pure virtual?